tsconfig.json

```diff
@@ -9,14 +9,32 @@
     "moduleResolution": "node",
     "baseUrl": "./",
     "paths": {
-      "garden-test-utils": ["utils/test/utilities.ts"]
+      "garden-test-utils": ["utils/test/utilities.ts"],
+      "@zendeskgarden/container-accordion": ["./packages/accordion/src/index.ts"],
+      "@zendeskgarden/container-breadcrumb": ["./packages/breadcrumb/src/index.ts"],
+      "@zendeskgarden/container-buttongroup": ["./packages/buttongroup/src/index.ts"],
+      "@zendeskgarden/container-field": ["./packages/field/src/index.ts"],
+      "@zendeskgarden/container-focusjail": ["./packages/focusjail/src/index.ts"],
+      "@zendeskgarden/container-focusvisible": ["./packages/focusvisible/src/index.ts"],
+      "@zendeskgarden/container-grid": ["./packages/grid/src/index.ts"],
+      "@zendeskgarden/container-keyboardfocus": ["./packages/keyboardfocus/src/index.ts"],
+      "@zendeskgarden/container-modal": ["./packages/modal/src/index.ts"],
+      "@zendeskgarden/container-pagination": ["./packages/pagination/src/index.ts"],
+      "@zendeskgarden/container-schedule": ["./packages/schedule/src/index.ts"],
+      "@zendeskgarden/container-scrollregion": ["./packages/scrollregion/src/index.ts"],
+      "@zendeskgarden/container-selection": ["./packages/selection/src/index.ts"],
+      "@zendeskgarden/container-splitter": ["./packages/splitter/src/index.ts"],
+      "@zendeskgarden/container-tabs": ["./packages/tabs/src/index.ts"],
+      "@zendeskgarden/container-tooltip": ["./packages/tooltip/src/index.ts"],
+      "@zendeskgarden/container-treeview": ["./packages/treeview/src/index.ts"],
+      "@zendeskgarden/container-utilities": ["./packages/utilities/src/index.ts"]
```

Comment:
> paths lets you declare how TypeScript should resolve an import in your require/imports.

I'm curious what were the consequences of not previously telling TS how to resolve imports. @jzempel do you know?

Reply:
If these are removed, the storybook `.stories.tsx` files show typescript errors. This same setup was done over on `react-components` a while ago so that we were able to directly import package exports into stories.

Reply:
I see, thanks!